src/main/java/seedu/address/model/procedure/DateWithoutTime.java

```diff
@@ -0,0 +1,73 @@
+package seedu.address.model.procedure;
```

Comment:
May I ask why you put this method under procedure package? 
Because it seems that Procedure doesn't really involve the use of DateWithoutTime. 
Rather, I feel this class helps to parse user input and provide a utility to help Procedures to be collected more easily under Clients.

Reply:
ah I see what u mean ok, maybe it goes better in the command package, since we will be using it to parse user input!

Reply:
You may consider commons package as well I think! Just my opinion 😅